tests/build.go

```diff
@@ -278,6 +278,10 @@ RUN echo foo > /bar`)
 	require.NoError(t, err, string(out))
 	require.True(t, buildDetailsPattern.MatchString(string(out)), fmt.Sprintf("expected build details link in output, got %q", out))
 
+	if isExperimental() {
+		t.Skip("build details link not displayed in experimental mode when build fails")
+	}
```

Comment:
Build details link is not displayed for a build that fails through controller: https://github.com/docker/buildx/actions/runs/8522265808/job/23342157852?pr=2376#step:7:771